src/model_exploration.py

```diff
@@ -0,0 +1,40 @@
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import classification_report
+from sentiment_exploration import *
+from textblob import TextBlob
+import xgboost as xgb
+
+
+def apply_vectorization(df):
+    # Initialize the vectorizer
+    vect = CountVectorizer(ngram_range = (1,2), max_features=10000, min_df = 2, stop_words = 'english')
+    return vect
```

Comment:
add type hints for all functions like this:

`def apply_vectorization(df: pd.DataFrame) -> CountVectorizer:
    # Initialize the vectorizer
    vect = CountVectorizer(ngram_range = (1,2), max_features=10000, min_df = 2, stop_words = 'english')
    return vect`

This allows you specify input and output types. Some types need to be imported:
`from typing import List, Dict, Pattern`